Jenkinsfile

```
@@ -1,5 +1,4 @@
 buildPlugin(useContainerAgent: true, configurations: [
-        [ platform: "linux", jdk: "11" ],
-        [ platform: 'linux', jdk: '17' ],
+        [ platform: 'windows', jdk: '17' ],
```

this plugin does not need windows testing, there is nothing that could be affected by it, and it just slows down builds massively
```suggestion
        [ platform: 'linux', jdk: '17' ],
```